app/scripts/modules/core/pipeline/config/triggers/git/gitTrigger.html

```diff
@@ -24,22 +24,22 @@
              ng-model="vm.trigger.project"
              name="project"
              required
-             placeholder="{{ displayText['vm.trigger.project'][vm.trigger.source] }}"
+             placeholder="{{ vm.displayText['vm.trigger.project'][vm.trigger.source] }}"
       />
     </div>
   </div>
   <div class="form-group">
     <label class="col-md-3 sm-label-right">
-      {{ displayText['pipeline.config.git.slug'][vm.trigger.source] }}
+      {{ vm.displayText['pipeline.config.git.slug'][vm.trigger.source] }}
       <help-field key="pipeline.config.git.slug"></help-field>
     </label>
     <div class="col-md-9">
       <input type="text" class="form-control input-sm"
              ng-model="vm.trigger.slug"
              name="slug"
              required
-             placeholder="{{ displayText['vm.trigger.slug'][vm.trigger.source] }}"
-             pattern="^((?!\:\/\/).)*$"
+             placeholder="{{ vm.displayText['vm.trigger.slug'][vm.trigger.source] }}"
+             pattern="^((?!://).)*$"
```

Comment:
Not sure if something changed with Chrome or what, but this was throwing an exception with the escaped slashes.